### PR TITLE
Benjamin/delete button(cancel booking)

### PR DESCRIPTION
**Pull request by Benjamin-GitHub**
ModalCancelBookingScreen created.

### CODE REVIEW

**Comment by Benjamin-GitHub** [comment]
There were some conflict, I did rebase my branch and now everrything seems fine. Also I checked factionality of App on my local machine

File: client/src/pages/ModalCancelBookingScreen.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+function ModalCancelBookingScreen({ booking, refreshBooking }) {
+	const [open, setOpen] = React.useState(false);
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+
+		fetch(`/api/bookings/${booking.id}`, {
+			method: "DELETE",
+			headers: {
+				"content-type": "application/jason",
```
Comment (gregdyke) [inline client/src/pages/ModalCancelBookingScreen.js]: application/json, not jason :)

**Review by gregdyke — DISMISSED** [review]
The PR still needs fixing/rebasing, otherwise everything looks great

File: client/src/pages/ModalCancelBookingScreen.js (hunk 1 above)
Comment (Benjamin-GitHub) [reply client/src/pages/ModalCancelBookingScreen.js]: Yeah! I see. Thanks

**Review by viktoriia-kosenko — APPROVED** [review]

**Pull request merged**